### PR TITLE
show_send_button:true parameter not working for me.This pull is a fix for that issue

### PR DESCRIPTION
**Pull request by golangbu**
fixing show_send_button:true issue.

This is my first pull request.If you find something wrong please delete this pull request.

### CODE REVIEW

**Comment by golangbu** [comment]
@jcbrand  I change files according to you.Please check

File: CHANGES.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 - #1188 Feature request: drag and drop file to HTTP Upload
 - #1268 Switch from SASS variables to CSS custom properties
 - #1278 Replace the default avatar with a SVG version
-
+- #1033 show send buuton:true issue fixes
```
Comment (jcbrand) [inline CHANGES.md]: Please fix the typo...

Comment (golangbu) [reply CHANGES.md]: @jcbrand . Sorry but what is typo. is there any spelling mistake

Comment (jcbrand) [reply CHANGES.md]: "typo" means typing mistake.

"buuton" should be "button".

Please change the text to: `#1033 Setting show_send_button to true didn't work`

**Comment by jcbrand** [comment]
Thanks @golangbu, can you also rebase your changes onto the latest master?

**Comment by jcbrand** [comment]
Thanks!

**Pull request merged**